src/common/transformations/tests/op_conversions/convert_avgpool_downgrade.cpp

```diff
@@ -0,0 +1,142 @@
+                                                                   true,
+                                                                   rounding_type,
+                                                                   ov::op::PadType::EXPLICIT);
+
+    avg_pool_v1->set_friendly_name("avg_pool_v1");
+
+    return std::make_shared<ov::Model>(avg_pool_v1->outputs(), ov::ParameterVector{input});
+}
+
+std::shared_ptr<ov::Model> create_exclude_pad_workaround_model() {
+    using ov::op::v0::Concat;
+    using ov::op::v0::Constant;
+    using ov::op::v1::ConvertLike;
+    using ov::op::v1::Pad;
+    using ov::op::v1::Subtract;
+    using ov::op::v3::Broadcast;
+    using ov::op::v3::ShapeOf;
+
+    const auto input = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::Shape{1, 3, 64, 64});
+    const ov::Strides strides{1, 1}, dilations{1, 1};
+    const ov::Shape pads_begin{1, 1}, pads_end{1, 1}, kernel{2, 2};
+
+    const auto zero = Constant::create(ov::element::f32, ov::Shape{}, {0});
+    const auto zero_node = std::make_shared<ConvertLike>(zero, input);
+    const auto zero_i64 = Constant::create(ov::element::i64, ov::Shape{}, {0});
+    const auto shape = std::make_shared<ShapeOf>(input, ov::element::i64);
+    const auto rank = std::make_shared<ShapeOf>(shape, ov::element::i64);
+    const auto pads_begin_node = Constant::create(ov::element::i64, ov::Shape{pads_begin.size()}, pads_begin);
+    const auto pads_end_node = Constant::create(ov::element::i64, ov::Shape{pads_end.size()}, pads_end);
+    const auto pads_len = Constant::create(ov::element::i64, ov::Shape{}, {pads_begin.size()});
+    const auto pads_diff = std::make_shared<Subtract>(rank, pads_len);
+    const auto pads_remaining = std::make_shared<Broadcast>(zero_i64, pads_diff);
+    const auto pads_begin_v1 =
+        std::make_shared<Concat>(ov::OutputVector{std::move(pads_remaining), std::move(pads_begin_node)}, 0);
+    const auto pads_end_v1 =
+        std::make_shared<Concat>(ov::OutputVector{std::move(pads_remaining), std::move(pads_begin_node)}, 0);
+    const auto pad_node =
+        std::make_shared<Pad>(input, pads_begin_v1, pads_end_v1, zero_node, ov::op::PadMode::CONSTANT);
+    const auto pads_begin_zeros = ov::Shape{0, 0};
+    const auto pads_end_zeros = ov::Shape{0, 0};
+    const auto avg_pool_v1 = std::make_shared<ov::op::v1::AvgPool>(input,
+                                                                   strides,
+                                                                   pads_begin_zeros,
+                                                                   pads_end_zeros,
+                                                                   kernel,
+                                                                   false,
+                                                                   ov::op::RoundingType::CEIL,
+                                                                   ov::op::PadType::EXPLICIT);
```

Comment:
Okay, looks like the test passed because the `pad_node` is not provided to the `AvgPool` in the expected graph as well.
```suggestion
    const auto pad_node =
        std::make_shared<Pad>(input, pads_begin_v1, pads_end_v1, zero_node, ov::op::PadMode::CONSTANT);
    const auto pads_begin_zeros = ov::Shape{0, 0};
    const auto pads_end_zeros = ov::Shape{0, 0};
    const auto avg_pool_v1 = std::make_shared<ov::op::v1::AvgPool>(pad_node,
                                                                   strides,
                                                                   pads_begin_zeros,
                                                                   pads_end_zeros,
                                                                   kernel,
                                                                   false,
                                                                   ov::op::RoundingType::CEIL,
                                                                   ov::op::PadType::EXPLICIT);
```

Reply:
Fixed.